src/services/fetchers/ByBitSpotFetcher.ts

```diff
@@ -50,67 +38,34 @@ class ByBitSpotFetcher {
       throw new Error(response.data.Message);
     }
 
-    const feedValues = this.resolveFeeds(inputs, response.data.result.list);
-
-    return feedValues ?? [];
+    return this.resolveFeeds(inputs, response.data.result.list);
   }
 
-  private resolveFeeds(inputs: ByBitSpotFetcherParams, priceList: Record<string, string>[]): OutputValue[] {
-    const outputs: OutputValue[] = [];
-    const priceDatas = priceList.filter((entry: Record<string, string>) => this.filterFeeds(entry));
+  private resolveFeeds(inputs: InputParams[], priceList: Record<string, string>[]): NumberOrUndefined[] {
+    const outputMap = new Map<string, NumberOrUndefined>();
 
-    this.logger.debug(`[ByBitSpotFetcher] Prices found: ${JSON.stringify(priceDatas)}`);
+    inputs.forEach((input) => {
+      outputMap.set(input.symbol, undefined); // Using the index as the value here for demonstration
+    });
 
-    priceDatas.forEach((priceData: Record<string, string>) => {
-      const feed = this.getInputFeed(priceData)!;
+    for (const price of priceList) {
+      const symbolRead = price.symbol;
```

Comment:
why we can not use simply `price.symbol`?

Reply:
We can. I think is more a style issue. I tend to dislike to repeate `price.symbol` on 4 different places and like to see the variable scope (in this case `symbolRead`, as short as possible.

Reply:
for me personally if we not having issue with line too long because we uisng lot of params in one line, it is better to not create clones of variables, this particular case I see no benefits except more code. but I'm ok if you want ot leave it.